app/pages/collections/list.cjsx

```diff
@@ -24,33 +38,73 @@ List = React.createClass
     Object.assign query, @props.location.query
 
     apiClient.type('collections').get query
 
   render: ->
     {location} = @props
-    <section className="resources-container collections-container">
+    classes = classNames {
+      "resources-container": true
+      "collections-container": true
+      "in-project-context": @props.project?
+    }
+
+    baseLink = if @props.project? then "/projects/#{@props.project.slug}/" else "/"
+    baseType = if @props.favorite then "favorites" else "collections"
+    shouldShowMoreLink = @props.params.profile_name? || @props.params.collection_owner?
+    showMoreLink = "#{baseLink}#{baseType}/"
+    shouldShowSubMenu = @props.project? && !@props.params.profile_name?
+    shouldShowZooWideLink = @props.project? && !shouldShowMoreLink
+    showZooWideLink = "/#{baseType}/"
+    shouldShowUserWideLink = @props.project? && shouldShowMoreLink
+    userForUserWideLink = if @props.params.profile_name? then @props.params.profile_name else @props.params.collection_owner
+    showUserWideLink = "/users/#{userForUserWideLink}/#{baseType}"
+
+    <section className={classes}>
+      {if shouldShowSubMenu
+        <CollectionsNav
+          translationObjectName="#{@props.translationObjectName}"
+          user={@props.user}
+          project={@props.project} />}
       <PromiseRenderer promise={@listCollections()}>{(collections) =>
         if collections?.length > 0
           meta = collections[0].getMeta()
           <div>
             <div className="resource-results-counter collection-results-counter">
-              {if meta
-                pageStart = meta.page * meta.page_size - meta.page_size + 1
-                pageEnd = Math.min(meta.page * meta.page_size, meta.count)
-                count = meta.count
-                <Translate pageStart={pageStart} pageEnd={pageEnd} count={count} content="#{@props.translationObjectName}.countMessage" component="p" />}
+              <p>
+                {if meta
+                  pageStart = meta.page * meta.page_size - meta.page_size + 1
+                  pageEnd = Math.min(meta.page * meta.page_size, meta.count)
+                  count = meta.count
+                  <Translate pageStart={pageStart} pageEnd={pageEnd} count={count} content="#{@props.translationObjectName}.countMessage" />}
+                {if shouldShowMoreLink
+                  <Link className="show-more" to="#{showMoreLink}">
+                    <Translate content="#{@props.translationObjectName}.more" />
+                  </Link>}
+                {if shouldShowZooWideLink
+                  <Link className="show-more" to="#{showZooWideLink}">
+                    <Translate content="#{@props.translationObjectName}.zooWide" />
+                  </Link>}
+                {if shouldShowUserWideLink
+                  <Link className="show-more" to="#{showUserWideLink}">
+                    <Translate content="#{@props.translationObjectName}.userWide" user="#{userForUserWideLink}"/>
```

Comment:
Is this the same point as the "does user need to be translated" query earlier? I explained in answer to that why passing the user to the translate function is the correct way to globalize a string containing a username - otherwise you have to algorithmically encode word order, which is language-specific. Or am i missing something?